docs/getting_started.md

```diff
@@ -130,7 +130,7 @@ Nor does it estimate any additional network time outside of EPP.
 `
 
 1. For payments the expected resolution time is 50-300 ms.
-2. For payments, if we have not reached a resolution within 6.6 seconds, we will return a 'AuthorisationFailed' callback. We will then perform a technical reversal immediately on the payment in question.
+2. For payments, if we have not reached a resolution within 6.6 seconds we will return a 'AuthorisationFailed' callback. We will then perform a technical reversal immediately on the payment in question.
```

Comment:
You probably wanted to use backticks around `AuthorisationFailed`?

Reply:
Thanks! Resolved :D